spotinst/ocean_aks_np_vm_sizes/fields_spotinst_ocean_aks_np_vm_sizes.go

```diff
@@ -310,6 +330,18 @@ func expandVmSizesFiltersList(data interface{}) ([]string, error) {
 	return result, nil
 }
 
+func expandGpuTypesFiltersList(data interface{}) ([]string, error) {
```

Comment:
this function is not required as we have a common function please re-use it

Reply:
changed and removed extra function